packages/contracts-bedrock/test/kontrol/proofs/OptimismPortal.k.sol

```diff
@@ -0,0 +1,110 @@
+pragma solidity ^0.8.13;
+
+import { DeploymentSummary } from "./utils/DeploymentSummary.sol";
+import { KontrolUtils } from "./utils/KontrolUtils.sol";
+import { Types } from "src/libraries/Types.sol";
+import {
+    OptimismPortalInterface as OptimismPortal,
+    SuperchainConfigInterface as SuperchainConfig
+} from "./interfaces/KontrolInterfaces.sol";
+
+contract OptimismPortalKontrol is DeploymentSummary, KontrolUtils {
+    OptimismPortal optimismPortal;
+    SuperchainConfig superchainConfig;
+
+    function setUp() public {
+        recreateDeployment();
+        optimismPortal = OptimismPortal(payable(OptimismPortalProxyAddress));
+        superchainConfig = SuperchainConfig(SuperchainConfigProxyAddress);
+    }
+
+    function test_kontrol_in_foundry(
```

Comment:
What's the difference between this test and `test_proveWithdrawalTransaction_paused`? They seem to be testing the same thing, but the UX of this test is much nicer because the function inputs use the structs/arrays directly, instead of having to craft them in the test

Reply:
The difference is in the test parameters, as you point out. Currently, Kontrol doesn't support symbolic `bytes` nor `bytes[]`, hence why we need to craft them in the test

Reply:
I see, so for now should delete the `function test_kontrol_in_foundry` test and add a `TODO comment` explaining how we can simplify the existing test once kontrol supports symbolic `bytes` and `bytes[]`